internal/modules/module.go

```diff
@@ -52,37 +53,51 @@ type Module struct {
 
 // getLatestVersion returns the latest version of a git repository
 // name should be a valid go import path, like github.com/<org>/<repo>
-func getLatestVersion(ctx context.Context, name string) (string, error) {
-	paths := strings.Split(name, "/")
+func getLatestVersion(ctx context.Context, tr *configuration.TemplateRepository) (string, error) {
+	paths := strings.Split(tr.Name, "/")
 
 	// github.com, getoutreach, stencil-base
 	if len(paths) < 3 {
-		return "", fmt.Errorf("invalid module path %q, expected github.com/<org>/<repo>", name)
+		return "", fmt.Errorf("invalid module path %q, expected github.com/<org>/<repo>", tr.Name)
 	}
 
 	gh, err := github.NewClient(github.WithAllowUnauthenticated())
 	if err != nil {
 		return "", err
 	}
 
+	if tr.Prerelease { // Use the newest, first, release.
```

Comment:
More of a question on the github api. GetLatestRelease will not return a latest RC?

Reply:
`GetLatestRelease` returns the latest _released_ version. https://docs.github.com/en/rest/releases/releases#get-the-latest-release